### ISSUE TITLE
Is there a LaunchBar equivalent of this addon?

### ISSUE THREAD

**Issue by arkhandir** [AnKingMed/Alfred-Anki-Search#1]
Some like Alfred, others like LaunchBar. Is there a LaunchBar equivalent of this addon?

**Comment by AnKingMed** [comment]
I don't think so. There is Alfred and keyboard maestro. If you make one, let me know and I'll link it here

**Comment by youtextme** [comment]
If the goal is Anki from a launcher and you’re weighing LaunchBar vs Alfred: also try free Raycast for a few days — there are Anki-related Store extensions, and you can keep Alfred/LaunchBar on your main hotkey while Raycast sits on Option-Space.

That way you compare Instant Send / workflow feel without buying another paid license first. Pay (Powerpack or LaunchBar) only after the daily Anki search path is clearly better in one of them.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.

**Comment by arkhandir** [comment]
The spammer reminded me that there is still no LaunchBar equivalent, so I created one:
https://github.com/arkhandir/Anki-Search
